tests/unit/test_bot_handlers_v2.py

```
@@ -195,34 +215,39 @@ async def test_schedule_command_authorized(mock_load_data, mock_update_context):
     assert "Monday" in call_args
     assert "Course 1" in call_args
 
+
 @pytest.mark.asyncio
-@patch('bot.handlers.chat_completion', return_value="This is a test answer.")
-@patch('bot.handlers.load_json_data', return_value=MOCK_STUDENTS)
-async def test_ask_command_authorized(mock_load_data, mock_chat_completion, mock_update_context):
+@patch("bot.handlers.chat_completion", return_value="This is a test answer.")
+@patch("bot.handlers.load_json_data", return_value=MOCK_STUDENTS)
+async def test_ask_command_authorized(
+    mock_load_data, mock_chat_completion, mock_update_context
```

Ruff (0.13.1)

Unused function argument: `mock_load_data`

(ARG001)